src/vscode/backend/LSP.ts

```diff
@@ -268,11 +268,11 @@ export class LSP extends EventEmitter<LSPEvents> {
         }))
     }
 
-    eval = async (text: string, pkgName: string, storeResult?: boolean): Promise<string | undefined> => {
+    eval = async (text: string, pkgName: string, storeResult?: boolean): Promise<string | Array<string> | undefined> => {
         try {
             const resp = await this.client?.sendRequest('$/alive/eval', { text, package: pkgName, storeResult })
 
-            if (!isObject(resp) || !isString(resp.text)) {
+            if (!isObject(resp) || (!isString(resp.text) && !isArray(resp.text, isString))) {
```

Comment:
Shouldn't it be this?

```suggestion
            if (!isObject(resp) || !isString(resp.text) || !isArray(resp.text, isString)) {
```

Reply:
I think we want to test that `resp.text` is not a string `AND` not an array of strings, because are both valid value. Otherwise if I receive an array of strings, the second condition would say `true` to `!isString(resp.text)` and return true, not processing the response.

<img width="530" alt="Screenshot 2025-02-06 at 10 29 45 am" src="https://github.com/user-attachments/assets/fec172aa-549b-447e-b581-064a3768b277" />